src/services/git/Git.ts

```diff
@@ -6,9 +6,11 @@ import { injectable } from 'inversify';
 import { ICache } from '../../scanner/cache/ICache';
 import { ErrorFactory } from '../../lib/errors/ErrorFactory';
 import { GitHubPullRequestState, GitHubFile, GitHubDir } from '../../services/git/IGitHubService';
+import * as nodePath from 'path';
+import { Metadata, MetadataType, IProjectFilesBrowserService } from '../model';
 
 @injectable()
-export class Git {
+export class Git implements IProjectFilesBrowserService {
   private repository: Repository;
   private gitHubClient: GitHubClient;
```

Comment:
I don't really like that this service is just for GitHub.

Reply:
So now, `Git` uses a `IContentRepositoryBrowser` instead of `GitHubClient`.